code-of-conduct-contact.md

```diff
@@ -0,0 +1,73 @@
+# Dat Ecosystem - Reporting Guide
+
+If you believe someone is violating [the code of conduct](./code-of-conduct.md) we ask that you report it to the Dat Ecosystem Consortium by emailing to `conduct@dat-ecosystem.org`.
+
+All reports will be kept confidential. In some cases we may determine that a public statement will need to be made.
+If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate law enforcement first.
+If you are unsure what law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
+
+If you are unsure whether the incident is a violation, or whether the space where it happened is covered by this Code of Conduct, we encourage you to still report it.
+We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation.
+
+We do not look negatively on you if we find the incident is not a violation.
+
+And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
+
+## In your report please include:
+
+- Your contact info _(so we can get in touch with you if we need to follow up)_
+- Names _(real, nicknames, or pseudonyms)_ of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+- When and where the incident occurred. Please be as specific as possible.
+- Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link and/or screenshot.
+- Any extra context you believe existed for the incident.
+- If you believe this incident is ongoing.
+- Any other information you believe we should have.
+
+
+## What happens after you file a report?
+
+You will receive an email from the dat consortium, acknowledging receipt immediately.
```

Comment:
dat consortium? not dat ecosystem? this might be old

also: what is the proper capitalisation of the group? i think it would be good to write somewhere & be consistent with in official documents :)

Reply:
The mail will be from the consortiu of "Dat Ecosystem". Updated in the last commit. Thanks for noticing!